recipes/r-rang/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set version = '0.2.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rang
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rang_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rang/rang_{{ version }}.tar.gz
+  sha256: 2e19eddc161d3d05406af27dc6d73c3bed40412d7b3c1be090f898fa372c9aab
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-fastmap
+    - r-httr
+    - r-jsonlite
+    - r-memoise
+    - r-parsedate
+    - r-pkgsearch
+    - r-remotes
+    - r-renv
+    - r-vctrs
+
+  run:
+    - r-base
+    - r-fastmap
+    - r-httr
+    - r-jsonlite
+    - r-memoise
+    - r-parsedate
+    - r-pkgsearch
+    - r-remotes
+    - r-renv
+    - r-vctrs
+
+test:
+  commands:
+    - $R -e "library('rang')"           # [not win]
+    - "\"%R%\" -e \"library('rang')\""  # [win]
+
+about:
+  home: https://github.com/chainsawriot/rang
+  license: GPL-3.0-only
```

CRAN shows `GPL (>=3)`, so...

```suggestion
  license: GPL-3.0-or-later
```